task/sql-tasks.js

```diff
@@ -259,17 +410,37 @@ async function task_1_19(db) {
  *
  */
 async function task_1_20(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT 
+        Employees.EmployeeID,
+        concat(FirstName, ' ', LastName) AS 'Employee Full Name',
+        sum(UnitPrice * Quantity) AS 'Amount, $'
+    FROM Employees 
+    JOIN Orders ON Employees.EmployeeID = Orders.EmployeeID 
+    JOIN OrderDetails ON Orders.OrderID = OrderDetails.OrderID
+    GROUP BY Employees.EmployeeID
+    ORDER BY sum(UnitPrice * Quantity) DESC
+    LIMIT 1;
+    `);
+    return result[0];
 }
-
 /**
  * Write a SQL statement to get the maximum purchase amount of all the orders.
  * | OrderID | Maximum Purchase Amount, $ |
  *
  * @return {array}
  */
 async function task_1_21(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT 
+        OrderID,
+        sum(UnitPrice * Quantity) AS 'Maximum Purchase Amount, $'
+    FROM OrderDetails
+    GROUP BY OrderID
+    ORDER BY sum(UnitPrice * Quantity) DESC

```

Comment:
Please, simplify ORDER BY clause using alias from prevuous step (don't forget escape quotes in sql query in node code)